modules/azure/governance-global/variables.tf

```diff
@@ -64,7 +64,8 @@ variable "resource_group_configs" {
       delegate_key_vault         = bool # Delegate KeyVault creation
       delegate_service_endpoint  = bool # Delegate Service Endpoint permissions
       delegate_service_principal = bool # Delegate Service Principal
+      disable_unique_suffix      = bool # Disable unique_suffix in name of key vault
```

Comment:
Not just key vault, should keep the description to be more generic for the future.